CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ SPDX-License-Identifier: CC-BY-SA-4.0
     - `{%slideshare user/my-awesome-presentation %}` -> Embedding removed
     - `{%speakerdeck foobar %}` -> Embedding removed
 - The use of `sequence` as code block language ([Why?](https://hedgedoc.org/faq/))
+- Comma-separated definition of tags in the yaml-metadata
```

Comment:
```suggestion
- Comma separated definition of tags in the yaml-metadata
```

Reply:
I don't think the hyphen is wrong there.
I just read [this article by grammarly](https://www.grammarly.com/blog/hyphen/) and actually we've got a two-word adjective in front of a noun here.
Also take a look into a dictionary: https://ende.dict.cc/?s=comma-separated

Reply:
Well.. then you choose. Feel free to ignore/revert the other hypen related comments.